pkg/vault/contracts/test/VaultAdminMock.sol

```diff
@@ -55,13 +60,37 @@ contract VaultAdminMock is IVaultAdminMock, VaultAdmin {
         IVault(address(this)).initializeBuffer(wrappedToken, amountUnderlying, amountWrapped, sharesOwner);
     }
 
+    function manualAddLiquidityToBuffer(IERC4626 wrappedToken, uint256 underlyingAmount, uint256 wrappedAmount) public {
+        bytes32 bufferBalances = _bufferTokenBalances[wrappedToken];
+
+        if (underlyingAmount > 0) {
+            IERC20(wrappedToken.asset()).transferFrom(msg.sender, address(this), underlyingAmount);
+            _reservesOf[IERC20(wrappedToken.asset())] += underlyingAmount;
+            // Issued shares amount = underlying amount.
+            _bufferTotalShares[wrappedToken] += underlyingAmount;
+            _bufferLpShares[wrappedToken][msg.sender] += underlyingAmount;
+        }
+        if (wrappedAmount > 0) {
+            IERC20(address(wrappedToken)).transferFrom(msg.sender, address(this), wrappedAmount);
+            _reservesOf[IERC20(address(wrappedToken))] += wrappedAmount;
+            uint256 issuedSharesAmount = wrappedToken.previewRedeem(wrappedAmount);
+            _bufferTotalShares[wrappedToken] += issuedSharesAmount;
+            _bufferLpShares[wrappedToken][msg.sender] += issuedSharesAmount;
+        }
```

Comment:
I don't think this math really adds up with the actual operation; is that intended? We're not considering total supply or rates when computing the shares. E.g. if we only add underlying amount, that will increase the shares without considering the existing liquidity, which doesn't add up.

Do we really need this method? If we do, I'd consider something even more 'obviously manual': just also send how many shares to mint to the sender as an argument, and add those to the total too.

EDIT: lol buffers are linear so this is actually fine. Feel free to leave like this.

Reply:
I just added this method because I need an easy way to add tokens to a buffer and leave it unbalanced so that I can test the rebalance. Since the `addLiquidityToBuffer` is proportional now, I don't have an easy way to do it (I could wrap/unwrap, but then I'd need to modify the tests, which wouldn't add any value to them).

I changed this method's name to `addLiquidityToBufferUnbalancedForTests`, which removes confusion about the actual operation and clarifies that it's only used for testing purposes.